### PR TITLE
refactor(e2e): Refactors locators and methods of Page Objects

### PR DESCRIPTION
**Pull request by Vere-Grey**
this first commit deals with dashboard and settings page object, rest will come in following commit but this one is already way too big (sorry)

- refactors locators, they are now defined as page object class properties -> better readability of methods, easier maintanance
- splits onboarding page object from dashboard one
- simplifies majority of methods, removes plenty of redundant one
- bunch of renaming to improve readbillity 

I have tried to move `completeOnboarding` step from tests to beforeAll but that cause electron app initialization before emulator is ready and device is not detected. Postponing that to a separate PR

### CODE REVIEW

File: packages/suite-desktop-core/e2e/support/pageActions/dashboardActions.ts [hunk 1]
```diff
@@ -1,72 +1,52 @@
...
 import { NetworkSymbol } from '@suite-common/wallet-config';
 
-import { waitForDataTestSelector } from '../common';
-
 export class DashboardActions {
     private readonly window: Page;
+    readonly discoveryBar: Locator;
+    readonly dashboardGraph: Locator;
+    readonly deviceSwitchingOpenButton: Locator;
+    readonly modal: Locator;
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/support/pageActions/dashboardActions.ts:10]: Way too general locator. I will deal with it as part of test refactoring.

File: packages/suite-desktop-core/e2e/support/pageActions/dashboardActions.ts [hunk 2]
```diff
@@ -1,72 +1,52 @@
...
-        await waitForDataTestSelector(this.window, '@dashboard/graph', { timeout: 30000 });
+        await this.discoveryBar.waitFor({ state: 'attached', timeout: 10_000 });
+        await this.discoveryBar.waitFor({ state: 'detached', timeout: 120_000 });
+        await expect(this.dashboardGraph).toBeVisible();
     }
 
     async openDeviceSwitcher() {
-        await this.window.getByTestId('@menu/switch-device').click();
-        const deviceSwitcherModal = this.window.getByTestId('@modal');
-        await deviceSwitcherModal.waitFor({ state: 'visible' });
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/support/pageActions/dashboardActions.ts:45]: .click() checks for visibility -> redundant check

File: packages/suite-desktop-core/e2e/support/pageActions/dashboardActions.ts [hunk 3]
```diff
@@ -1,72 +1,52 @@
...
+        await this.modal.waitFor({ state: 'detached' });
         await this.discoveryShouldFinish();
     }
-
-    // asserts
-    async assertHasVisibleBalanceOnFirstAccount(network: NetworkSymbol) {
-        const locator = this.window.getByTestId(`@wallet/coin-balance/value-${network}`).first();
-
-        await expect(locator).toBeVisible();
-    }
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/support/pageActions/dashboardActions.ts:71]: replaced by expect on test level

File: packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts [hunk 4]
```diff
@@ -1,85 +1,98 @@
...
+        await expect(this.debugTabButton).toBeVisible();
     }
 
-    async goToSettingSection(section: Section) {
-        await this.window.getByTestId(`@settings/menu/${section}`).click();
-        //TODO: #15552 Refactor navigation verification to specific tests and remove this method completely
-        await this.window
-            .getByTestId(settingSectionsLocators[section])
-            .waitFor({ state: 'visible', timeout: 10_000 });
-    }
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts:37]: I have spent like 30 minute thinking whether keep it or not. But in the end the implemenetation of this method grow way too much due to locator rafactor. 
Anyway every test that does the navigation, then interact with element on the page. So the verification was duplicate and was checking just one random element on that section. So I completely replaced it with just click on tab on test level

File: packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts [hunk 5]
```diff
@@ -1,85 +1,98 @@
...
+    readonly applicationTabButton: Locator;
+    readonly deviceTabButton: Locator;
+    readonly coinsTabButton: Locator;
+    readonly earlyAccessJoinButton: Locator;
+    readonly earlyAccessConfirmCheck: Locator;
+    readonly earlyAccessConfirmButton: Locator;
+    readonly earlyAccessSkipButton: Locator;
+    readonly settingsCloseButton: Locator;
+    readonly modal: Locator;
+    //coin Advance settings
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts:19]: I had quite a pickle with naming of these. Originally the naming was refering to networks. I kept that somewhere but change a lot to coins. Thanks to small discussion with Boso.
Please feel free to suggest alternatives. As I have still very limited knowledge of the product and its naming.

File: packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts [hunk 6]
```diff
@@ -1,85 +1,98 @@
...
+    readonly debugTabButton: Locator;
+    readonly applicationTabButton: Locator;
+    readonly deviceTabButton: Locator;
+    readonly coinsTabButton: Locator;
+    readonly earlyAccessJoinButton: Locator;
+    readonly earlyAccessConfirmCheck: Locator;
+    readonly earlyAccessConfirmButton: Locator;
+    readonly earlyAccessSkipButton: Locator;
+    readonly settingsCloseButton: Locator;
+    readonly modal: Locator;
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts:18]: Again, a way too general locator. Postponing to another refactor

File: packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts [hunk 7]
```diff
@@ -1,85 +1,98 @@
...
-        await this.window.getByTestId('@settings/early-access-join-button').click();
-        const eapModal = this.window.getByTestId('@modal');
-        await eapModal.waitFor({ state: 'visible' });
-        await eapModal.getByTestId('@settings/early-access-confirm-check').click();
-        await eapModal.getByTestId('@settings/early-access-confirm-button').click();
-        await eapModal.getByTestId('@settings/early-access-skip-button').click();
-    }
-
-    async getEarlyAccessButtonText() {
-        return await this.window.getByTestId('@settings/early-access-join-button').textContent();
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts:78]: Removed and replace by a oneline on test level

File: packages/suite-desktop-core/e2e/tests/general/eap-modal.test.ts [hunk 8]
```diff
@@ -11,13 +11,14 @@ import { test, expect } from '../../support/fixtures';
 // TODO: #15561 FIX settings cleanup:  eap setting is remembered even after cache cleanup at the beginning of the test. This shouldn't affect gha run but breaks the local one.
 test.skip(!process.env.GITHUB_ACTION, 'Test is working only in CI. Skipping local run.');
 
-test('Join early access button', async ({ dashboardPage, topBar, settingsPage }) => {
-    const buttonText = 'Leave';
+test.beforeAll(async ({ onboardingPage }) => {
+    await onboardingPage.completeOnboarding();
+});
```
Comment (Vere-Grey) [inline packages/suite-desktop-core/e2e/tests/general/eap-modal.test.ts:16]: I tried to move this step to before in all tests but it didnt work in tests with emulators. So I will do the rest once I move emulator to fixtures

**Review by HajekOndrej — APPROVED** [review]

**Pull request merged**